src/core/lib/tsi/ssl_transport_security.h

```diff
@@ -137,7 +136,8 @@ tsi_result tsi_create_ssl_server_handshaker_factory(
     const size_t *pem_private_keys_sizes, const unsigned char **pem_cert_chains,
     const size_t *pem_cert_chains_sizes, size_t key_cert_pair_count,
     const unsigned char *pem_client_root_certs,
-    size_t pem_client_root_certs_size, int force_client_auth,
+    size_t pem_client_root_certs_size,
+    tsi_client_certificate_request_type client_certificate_request,
```

Comment:
Shouldn't this be okay given that this is internal to gRPC and is used only in C/C++ code? I am okay with adding an another method similar to grpc_ssl_server_credentials_create_ex if you feel strongly about this.